app/controllers/main/routes.py

```diff
@@ -1,5 +1,7 @@
 from flask import request, render_template, g, abort, flash, redirect, url_for
 import datetime
+import re
```

Comment:
I don't see where you use regular expressions

Reply:
Have removed it fixed